app/views/catalog/_previous_next_doc.html.erb

```diff
@@ -1,2 +1,2 @@
-<% Deprecation.warn(self, 'The partial _previous_next_doc.html.erb will be removed in 8.0. Render Blacklight::SearchContextComponent instead.') %>
-<%= render(Blacklight::SearchContextComponent.new(search_context: @search_context, search_session: search_session)) %>
```

Comment:
Is this a backwards-compatible change if a downstream app either overrode or provided a custom template for the `Blacklight::SearchContextComponent` class?

Reply:
Fixed - you're right, we should keep using old component name